app/Controller/CoGroupsController.php

```diff
@@ -764,49 +763,59 @@ public function beforeRender()
     parent::beforeRender();
   }
 
+  /**
+   * Determine the conditions for pagination of the index view, when rendered via the UI.
+   *
+   * @since  COmanage Registry v3.1.1
+   * @return Array An array suitable for use in $this->paginate
+   */
+
   /**
    * Determine the conditions for pagination of the index view, when rendered via the UI.
    *
    * @since  COmanage Registry v0.8
    * @return Array An array suitable for use in $this->paginate
    */
-  
+
   public function paginationConditions() {
-    $fn = "CoGroupsController@paginationConditions";
+    $fn = "paginationConditions";
     $this->log(get_class($this)."::{$fn}::@ ", LOG_DEBUG);
     $pagcond = array();
-    
+
     // Set page title
     $this->set('title_for_layout', _txt('fd.co_people.search'));
     // Use server side pagination
-    
+
     if($this->requires_co) {
       $pagcond['conditions']['CoGroup.co_id'] = $this->cur_co['Co']['id'];
     }
-    
-    // Filtering by name operates using any name, so preferred or other names
-    // can also be searched. However, filter by letter ("familyNameStart") only
-    // works on PrimaryName so that the results match the index list.
-    
+
     // Filter by name
     if(!empty($this->params['named']['Search.name'])) {
       $searchterm = strtolower($this->params['named']['Search.name']);
       // We set up LOWER() indices on these columns (CO-1006)
-      $pagcond['conditions']['LOWER(Name.given) LIKE'] = "%$searchterm%";
+      $pagcond['conditions']['LOWER(CoGroup.name) LIKE'] = "%$searchterm%";
     }
-    
+
     // Filter by description
     if(!empty($this->params['named']['Search.desc'])) {
       $searchterm = strtolower($this->params['named']['Search.desc']);
-      $pagcond['conditions']['LOWER(Name.family) LIKE'] = "%$searchterm%";
+      $pagcond['conditions']['LOWER(CoGroup.description) LIKE'] = "%$searchterm%";
     }
-    
-    // Filter by category (ONLY for LOFAR)
-    if(!empty($this->params['named']['Search.category'])) {
-      $searchterm = strtolower($this->params['named']['Search.category']);
-      $pagcond['conditions']['LOWER(PrimaryName.family) LIKE'] = "$searchterm%";
+
+    // LOFAR Community filtering support
```

Comment:
How it this connected with LOFAR ?
what is the actual change ?
what is the added value of this change support for what ?

Reply:
Couldn't this change be generalised for the use by other communities ?

Reply:
LOFAR requested specific search fields beyond the ones provided by COManage. This is why we extended the functionality so as to much their requirements. Nevertheless, this is not something that can be generalized since it servers specific LOFAR name conventions.
Since this is pull request is closed any additional comment should be made on the open one.